lib/absinthe/pipeline.ex

```diff
@@ -92,6 +92,7 @@ defmodule Absinthe.Pipeline do
       # Prepare for Execution
       Phase.Document.CascadeInvalid,
       Phase.Document.Flatten,
+      {Phase.Document.Complexity, options},
```

Comment:
Can we just support a `nil` default, making the phase "installed" by default, but essentially no-op?